internal/blueprint/blueprint.go

```diff
@@ -66,7 +67,14 @@ func ListOutputFormats() []string {
 // ToPipeline converts the blueprint to a pipeline for a given output format.
 func (b *Blueprint) ToPipeline(outputFormat string) (*pipeline.Pipeline, error) {
 	if output, exists := outputs[outputFormat]; exists {
-		return output.translate(b), nil
+		p := output.translate(b)
+		if b.Customizations != nil {
```

Comment:
I think you should leave this to each translation function to deal with. I don't think you need to change this function at all. For two reasons: the stages you append should perhaps not go last (SELinux probably should). and it might be that a customization changes some stage, rather than just append to it. As this is going to be in conflict with Martin's work, I suggest simply leaving it out for now, and then adding the same call `addToPipeline()` to each of the translations as he finishes them in follow-up PRs. If it turns out that we need to split addToPipeline() into smaller helper functions for some output  formats that's totally fine of course, but could also be done in follow-ups.